demisto_sdk/commands/common/logger.py

```diff
@@ -328,6 +328,8 @@ def logging_setup(
 
     set_demisto_logger(demisto_logger)
 
+    cli_args = [f'"{arg}"' if " " in arg else arg for arg in sys.argv[1:]]  # Add quotes to spaced arguments
```

Comment:
`[1:]` because 0th is the path/python? I can imagine a situation where it could be useful. How dirty is it if we don't slice?

Reply:
When running from the repository, `argv[0]` is `/Users/username/dev/demisto/demisto-sdk/demisto_sdk/__main__.py`

When running from a compiled package (I installed from a test branch):
`/Users/username/Library/Caches/pypoetry/virtualenvs/demisto-sdk-686ZqX68-py3.10/bin/demisto-sdk`

We can do `Path(arg).name`, but I don't see the benefit of that, and we'll get `__main__.py` when running from code (not sure if that's good or bad, don't see a case where it matters, honestly)

Reply:
the python path isn't a bad thing to have in the logs, we have users working with multiple envs 
If you rather keep this log line relatively short, consider logging the `[0]` in a separate line